doc/api/http.md

```diff
@@ -142,7 +142,7 @@ changes:
     header is always sent when using an agent except when the `Connection`
     header is explicitly specified or when the `keepAlive` and `maxSockets`
     options are respectively set to `false` and `Infinity`, in which case
-    `Connection: close` will be used. **Default:** `false`.
+    `Connection: close` will be used. **Default:** `true`.
```

Comment:
https://github.com/nodejs/node/blob/99f6084ef04dd868c7b894ca0fdbb5088773ca6c/lib/_http_agent.js#L111

The default value is `false`.

Reply:
:thinking: 

So a another line in doc is misleading.

https://github.com/nodejs/node/pull/51115/files#diff-d692ac4524379ec6a1201165e8ff8d3267c8130e07014e8221ebf7e6f80c6641R183-R184

Because https://github.com/nodejs/node/blob/99f6084ef04dd868c7b894ca0fdbb5088773ca6c/lib/_http_agent.js#L557

I understand the default value for a new Agent is falsy keepAlive, but the "default" globalAgent is set to true.

```
~ » nvm use system
Now using system version of node: v20.10.0 (npm v10.2.5)
--------------------------------------------------------------------------------------------------
~ » node -pe 'const {keepAlive, keepAliveMsecs} = http.globalAgent; ({keepAlive, keepAliveMsecs})'
{ keepAlive: true, keepAliveMsecs: 1000 }
```

```
~ » nvm use 18
Now using node v18.19.0 (npm v10.2.3)
--------------------------------------------------------------------------------------------------
~ » node -pe 'const {keepAlive, keepAliveMsecs} = http.globalAgent; ({keepAlive, keepAliveMsecs})'
{ keepAlive: false, keepAliveMsecs: 1000 }
```

___

To be complete, in our organization (https://www.radiofrance.fr/) we had some surprises with this change in our kubernetes infrastructure.

Reply:
Yes, you are right: the misleading line is the one you linked above. In #43522 I changed some options for the default agent (while leaving the constructor behavior untouched).
Could you please update this PR to fix that offending line instead of the one you proposed?

Reply:
Done. I hope my changes are what you expect.